ui/src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
+import CreatePoll from './pages/CreatePoll';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
```

Comment:
App.js seems to be a contentious area for merge conflicts and I know why; because we don't have React Router implemented yet!
React Router basically allows for dynamic loading of containers depending on what route you are in the SPA. We still haven't used that, hence all the conflicts in App.js
I've been waiting on PR #11 to be merged since that contains React-Router, so either we wait for #11 to merge or you can reimplement the react-router at this level again. I say wait a bit more, hopefully we get it through.